lib/matplotlib/backends/backend_qt4agg.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, figure):
         if DEBUG:
             print('FigureCanvasQtAgg: ', figure)
         FigureCanvasQT.__init__(self, figure)
+        FigureCanvasQTAggBase.__init__(self, figure)
```

Comment:
With MEP27 we begin to introduce super into the backends, as we work to standardise the backends.  Currently we do it for all of the GUI elements, apart from the canvas.  ``Wx`` has an odd call signature for the constructor which makes ``super`` in FigureCanvas impossible until I have that fixed on master.  I do have it fixed in my Wx branch of MEP27 but naturally that will wait until we have the main MEP27 PR into master.